packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -194,5 +194,7 @@ export default (async (_, __, options) => {
     }
   };
 
-  printFixOptions({onKeyPress});
+  if (!stats.errors && stats.warnings) {
```

Comment:
shouldn't this be `if (stats.errors || stats.warnings)`?

Reply:
Oh yeah, sorry, this is totally wrong 😅

End of the day 🤷‍♂️

Reply:
All fixed 🙂